app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ Required ] I think it would be better if your design for the login page matches the design on the project wireframe. Currently, your design differs from the project wireframe in the following areas.
   
   - [ ]  The login form should be centered in the middle of the page, currently, the form in your design is not centered.
   - [ ]  The `Login` button and the `input` elements should have the same width just the way it is in the project wireframe. Please see the screenshot below for more details on this.


## Your login form design
![Screenshot from 2023-07-07 01-23-52](https://github.com/muhammedb8826/blog-app/assets/16392046/65723f22-d8d3-4a41-9897-36552d6aee30)


## Project wireframe design
![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)